packages/components/src/interfaces/complex/LogoCloud.ts

```diff
@@ -0,0 +1,39 @@
+import type { Static } from "@sinclair/typebox"
+import { Type } from "@sinclair/typebox"
+
+export const LogoCloudSchema = Type.Object(
+  {
+    type: Type.Literal("logocloud", { default: "logocloud" }),
+    images: Type.Array(
+      Type.Object({
+        src: Type.String({
+          title: "Upload image",
+          format: "image",
+        }),
+        alt: Type.String({
+          title: "Alternate text",
+          maxLength: 120,
+          description:
+            "Add a descriptive alternative text for this image. This helps visually impaired users to understand your image.",
+        }),
+      }),
+      {
+        title: "Images for logo cloud",
+        minItems: 1,
+        maxItems: 10,
+      },
+    ),
+    title: Type.String({
+      title: "Title for the logo cloud",
+      description: "Upload the images for the logo cloud here or provide a url",
```

Comment:
is this correct? seems like this `title` is more for what the content will display?

Reply:
this matches the header in our edit page! to give you an example, this is the `label` for statistics! i didn't put too much thought into this cos we're not allowing studio editing for logo cloud 
![image](https://github.com/user-attachments/assets/24e2df12-5f9d-4510-a6df-062b832c7a12)
<img width="433" alt="image" src="https://github.com/user-attachments/assets/1535035b-fcce-4c61-ac87-94b3ffcec4d2" />

Reply:
in this case, perhaps we can remove it so we won't have an issue of not remembering to fix this in the future when we support on studio?

Reply:
i added this here so it's less work next time when we want to implement it in studio ;--;